code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -450,19 +450,60 @@ The proc would return a human next to the bot to be set to the patient var.
 Pass the desired type path itself, declaring a temporary var beforehand is not required.
 */
 /mob/living/simple_animal/bot/proc/scan(atom/scan_type, atom/old_target, scan_range = DEFAULT_SCAN_RANGE)
-	var/final_result
-	for(var/scan in shuffle(view(scan_range, src))) //Search for something in range!
-		var/atom/A = scan
-		if(!istype(A, scan_type)) //Check that the thing we found is the type we want!
-			continue //If not, keep searching!
-		if((A.UID() in ignore_list) || (A == old_target) ) //Filter for blacklisted elements, usually unreachable or previously processed oness
+	var/turf/T = get_turf(src)
+	if(!T)
+		return
+	var/list/adjacent = T.GetAtmosAdjacentTurfs(1)
```

Comment:
```suggestion
	var/list/adjacent = T.GetAtmosAdjacentTurfs(TRUE)
```

Reply:
Ah, I didn't check what the 1 stood for. Since it's "adjacent" I thought it meant "tiles in a 1 tile radius".